plaso/cli/status_view.py

```diff
@@ -389,14 +393,18 @@ def SetMode(self, mode):
     """
     self._mode = mode
 
-  def SetSourceInformation(self, source_path, source_type, filter_file=None):
+  def SetSourceInformation(
+      self, source_path, source_type, artifact_filters=None,
+      filter_file=None):
     """Sets the source information.
 
     Args:
       source_path (str): path of the source.
       source_type (str): source type.
+      artifact_filters (Optional[str]): artifact filters.
```

Comment:
please reword, maybe something like: names of the artifact definitions to use as filter.

Reply:
Done